### DISCUSSION TITLE
Metadata on non-traditional deposits

### DISCUSSION THREAD

**Discussion by bjr70** [MESH-Research/knowledge-commons-repository#108] [General]
We're getting into some gray metadata territory - blog posts. They generally don't have persistent identifiers on the blog itself, and is unlikely to have funding. Check the sheet here: https://docs.google.com/spreadsheets/d/1p8Prxpvfr2GwhQlJjo5W_wAwkyno5kxKWIfIvJBRU2g/edit?usp=sharing. Thoughts?

**Comment by mikethicke** [comment]
I think you'd want the URL of the blog post and maybe the URL of the blog?

**Reply by bjr70** [reply]
I had the blog URL but added blog post URL.

**Comment by bjr70** [comment]
I'm working through the metadata fields for the resource types. See the ones I've done so far here: https://drive.google.com/drive/folders/1X8E8Gn-UOxfFl19bdNGULRAG7CmU-nRo?usp=drive_link

**Comment by rekmla** [comment]
Comment on blog post metadata to support citation:
How do you cite a blog post in MLA?
[image: How to Cite a Blog Post in MLA | EasyBib Citations]
MLA Blog Post Citation Structure:

Author's Last Name, Author's First Name. “Title of Post.” Blog Name,
Publisher (only include this information if it is different than the name
of the blog site), date blog post was published, URL. Column or section
name (if applicable).

On Thu, Jun 22, 2023 at 9:53 AM Bonnie Russell ***@***.***>
wrote:

> I'm working through the metadata fields for the resource types. See the
> ones I've done so far here:
> https://drive.google.com/drive/folders/1X8E8Gn-UOxfFl19bdNGULRAG7CmU-nRo?usp=drive_link
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/MESH-Research/knowledge-commons-repository/discussions/108#discussioncomment-6252462>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB7ZYVWSWFUAPT3VOCRRUGTXMRE4LANCNFSM6AAAAAAZG4KARE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <MESH-Research/knowledge-commons-repository/repo-discussions/108/comments/6252462
> @github.com>
>


-- 
R. Eric Knappe
Head of Software Development
Modern Language Association

www.mla.org

**Reply by bjr70** [reply]
Thanks @rekmla! Super helpful.

**Comment by kfitz** [comment]
This might warrant a new discussion, but it's connected here: Citation managers like Zotero are able to pull metadata from a range of digital publications directly. They're not perfect -- sometimes they need correction or augmentation. But are there things that we could (at some point -- perhaps not for first release) borrow from Zotero such that a user could begin a deposit by URL and have what metadata is available pulled and presented to the depositor for accept/correct/reject?

**Reply by bjr70** [reply]
That might work for previously published work, not sure how that would work for non-published work. Definitely something to think through and add to a list of possible future features. I love the idea if we could figure out how to get it to work. In thinking about CORE a lot of the work deposited is not previously published, so that's a consideration. That could change with the next version and the ability to create metadata-only deposits, though. I'd love to think through this more as we get into building a list of future features.

**Reply by kfitz** [reply]
Oh yeah, for sure: something would have to be published at a URL in order for deposit-by-URL to work. But a blog post, for instance, which is what made me think of this: if I click the "add to Zotero" button in my browser, Zotero sucks in the major metadata categories needed to produce a citation. If we could take advantage of that in our deposit workflow for stuff that's been published online, it could be huge.

**Reply by bjr70** [reply]
Ooh good point! I didn't think of blog posts when I wrote that. I'm going to record that as a future feature request.

**Reply by bjr70** [reply]
I've added an epic on the KCR board for future feature requests and made an issue and stuck it in the icebox for now.

Epic: https://app.zenhub.com/workspaces/knowledge-commons-repository-6419d860f16c9b3d0840f040/issues/gh/mesh-research/knowledge-commons-repository/127
